tests/imageupload_test.go

```diff
@@ -0,0 +1,106 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/virtctl/imageupload"
+	"kubevirt.io/kubevirt/tests"
+)
+
+const (
+	uploadProxyService   = "svc/cdi-uploadproxy"
+	uploadProxyNamespace = "kube-system"
```

Comment:
We have a `kubevirtInstallNamesplace` pflag. Can you use that instead?

Reply:
Didn't know about that.  Will do!